docs/section16/Rule 16-5.md

```diff
@@ -0,0 +1,38 @@
+# Elevators – Rule 16-5  
+**Schema Version:** 0.0.36        
+**Mandatory Rule:** True          
+**Rule ID:** 16-5  
+**Rule Description:** When included in the proposed design, the baseline elevator cab ventilation fan and lights shall operate continuously  
+**Rule Assertion:** B-RMD = Expected Value                                           
+**Appendix G Section:** G3.1  
+**Appendix G Section Reference:** Table G3.1-16 Baseline Building Performance  
+**Data Lookup:** None  
+**Evaluation Context:** Each elevator  
+
+**Applicability Checks:**  
+  1. The proposed building has an elevator.  
+
+**Function Call:**  
+compare_schedules
+
+**Applicability Check 1:**
+- Rule is applicable if any building in the proposed RMD has an elevator: `if any(len(building.elevators) > 0 for building in P_RMD...buildings:`
+
+## Rule Logic:
+- Determine if the project schedule uses a leap year: `is_leap_year = RPD.schedule.is_leap_year`
+- If the project schedule uses a leap year, set total hours to 8784, else 8760: `total_hours = 8784 if is_leap_year else 8760`
+- Create a continuous schedule to compare against, and also to use as the mask schedule for the function: `continuous_schedule = [1]*total_hours`
+- For each elevator in the baseline RMD: `for elevator in B_RMD...elevators`
+  - Get the modeled schedule for the baseline elevator cab ventilation fan: `cab_ventilation_schedule_b = elevator.cab_ventilation_fan_multiplier_schedule`
+  - Get the modeled schedule for the baseline elevator cab lighting: `cab_lighting_schedule_b = elevator.cab_lighting_multiplier_schedule`
+  - Compare the cab ventilation schedule with the continuous schedule: `vent_sched_compare_data = compare_schedules(cab_ventilation_schedule_b, continuous_schedule, continuous_schedule, is_leap_year)`
+  - Compare the cab lighting schedule with the continuous schedule: `light_sched_compare_data = compare_schedules(cab_lighting_schedule_b, continuous_schedule, continuous_schedule, is_leap_year)`
```

Comment:
not sure what the 'continuous schedule' is .. I believe thee intent is to verify that ventilation is provided when a zone is expected to be occupied .. Maybe we can look at the building open/close hrs and verify that the ventilation schedule is set to 1 for the open/close hrs?

Reply:
Appendix G just says the ventilation fan and lighting "both operate continuously" and I was not comfortable interpreting that to mean during building operating hours only in the implementation

Reply:
This makes sense